### PR TITLE
use latest security-library

### PR DESCRIPTION
**Pull request by etwillbefine**
@rebuy-de/prp-rebuy-silo-archetype

### CODE REVIEW

**Comment by peterpunch** [comment]
👍

**Comment by AlexanderHagenhoff** [comment]
@etwillbefine I can't see the version 5.1.1 [here](https://github.com/rebuy-de/java-library/releases) 
What am I doing wrong?

**Comment by bjoernhaeuser** [comment]
because the release is broken and called spring-sec-5.1.1 :-/

**Comment by AlexanderHagenhoff** [comment]
at least the version is found

...so 👍

File: src/main/resources/archetype-resources/silo/src/main/java/silo/configuration/ClientConfiguration.java [hunk 1]
```diff
@@ -31,13 +31,11 @@
...
     public RemoteTokenServices remoteTokenServices(RemoteTokenServicesSettings remoteTokenServicesSettings)
     {
-        RemoteTokenServicesConfig config = new RemoteTokenServicesConfig(
-            remoteTokenServicesSettings.clientId,
-            remoteTokenServicesSettings.secret,
-            remoteTokenServicesSettings.endpoint
-        );
-
-        return new RemoteTokenServicesBuilder().build(config);
+        return new RemoteTokenServicesBuilder()
```
Comment (AlexanderHagenhoff) [inline src/main/resources/archetype-resources/silo/src/main/java/silo/configuration/ClientConfiguration.java]: you have to fix the import for this ... @etwillbefine

**Comment by AlexanderHagenhoff** [comment]
sorry 👎 

you have to fix the import
import com.rebuy.library.security.builder.RemoteTokenServicesBuilder;
instead of
import com.rebuy.library.security.service.RemoteTokenServicesBuilder;

**Comment by AlexanderHagenhoff** [comment]
and remember to do a release of the archetype as soon as this is done please

**Comment by AlexanderHagenhoff** [comment]
This stuff  import com.rebuy.library.security.configuration.RemoteTokenServicesConfig seems to be gone too

**Comment by etwillbefine** [comment]
@AlexanderHagenhoff sorry fixed.

**Comment by AlexanderHagenhoff** [comment]
👍 good to go

**Comment by bjoernhaeuser** [comment]
@deterministic-arts can you please take care of releasing a proper 5.1.0 of the security lib? Please also remove the broken release afterwards.

**Pull request merged**